src/Checker/AbstractChecker.php

```diff
@@ -11,11 +11,26 @@
 
 abstract class AbstractChecker implements CheckerInterface
 {
+    /**
+     * @var array<mixed>
+     */
+    protected array $config = [];
+
     /**
      * @return bool
      */
     public function isApplicable(): bool
     {
         return true;
     }
+
+    /**
+     * @param array<mixed> $config
+     *
+     * @return void
+     */
+    public function setConfig(array $config): void
```

Comment:
Don't need pass configuration like that.
1. Services should be stateless
2. Inconsistent behavior

Put it in interface:
`public function check(CheckerInputDataDto $inputData, array $configuration = []): CheckerResponseDto;`

Reply:
Moved the configuration to CheckerInputDataDto